flink-table-store-common/src/main/java/org/apache/flink/table/store/data/BinaryRow.java

```diff
@@ -49,9 +50,14 @@
  * bigger pageSize of MemorySegment.
  *
  * <p>Variable-length part may fall into multiple MemorySegments.
+ *
+ * @since 0.4.0
```

Comment:
Also add this to `BinaryArray`, `BinaryMap`, `ColumnarRow` and many other classes?

Reply:
Added. ColumnarRow is still internal.